dotcom-rendering/playwright/lib/cmp.ts

```diff
@@ -0,0 +1,74 @@
+import type { BrowserContext, Page } from '@playwright/test';
+import { waitForIsland } from './islands';
+
+const SP_LAYER1_IFRAME = '[id*="sp_message_iframe"]';
+const SP_LAYER1_ACCEPT_ALL_BUTTON = 'button.sp_choice_type_11';
+const SP_LAYER2_MANAGE_MY_COOKIES_BUTTON = 'button.sp_choice_type_12';
+
+const SP_LAYER2_IFRAME = 'iframe[title="SP Consent Message"]';
+const SP_LAYER2_ACCEPT_ALL_BUTTON = 'button.sp_choice_type_ACCEPT_ALL';
+const SP_LAYER2_REJECT_ALL_BUTTON = 'button.sp_choice_type_REJECT_ALL';
+
+/**
+ * Accept all on the Sourcepoint CMP banner
+ * @param page
+ */
+const cmpAcceptAll = async (page: Page): Promise<void> => {
+	const acceptAllButton = page
+		.frameLocator(SP_LAYER1_IFRAME)
```

Comment:
This is how you get a [locator](https://playwright.dev/docs/api/class-locator) to an iframe in Playwright:
https://playwright.dev/docs/api/class-framelocator